resources/lang/en/thoughts/alive/general.json

```
@@ -5494,6 +5494,276 @@
             "living"
         ]
     },
+    {
+        "id": "allergies_gen",
+        "thoughts": [
+            "Is having a sneezing fit",
+            "Is having an extra sniffily day",
+            "Is having a non-sniffily day",
```

```suggestion
            "Is having a no-sniffles day",
```
Sounds a bit more natural